deploy/helm/dynatrace-oneagent-operator/templates/Common/application.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: dynatrace-oneagent-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    dynatrace: operator
+    operator: oneagent
+spec:
+  descriptor:
+    type: "Dynatrace OneAgent Operator"
+    version: "0.1.0"
```

Comment:
This version is?

Reply:
I think for helm

Reply:
This version defines the version of the chart - the appversion defines the version of the app which is used

Reply:
Shouldn't both be the same? I'm aware they don't need to match, it's more like, if a customer installs this Helm Chart 0.1.0, which Operator version should they expect?

Reply:
Good argument. But I doubt that we are going to maintain `appVersion` for each release. Do we?

Reply:
Customer will get the one which is in values file. It contains the image to use.